src/redux/books/books.js

```diff
@@ -0,0 +1,48 @@
+// Action Types
+export const ADD_BOOK = 'books/books/ADD_BOOK';
+export const REMOVE_BOOK = 'books/books/REMOVE_BOOK';
+
+// Initial State
+const initialState = {
+  books: [],
+  isLoading: false,
+  getId: null,
+};
+
+// Reducers
+const bookReducer = (state = initialState, action) => {
+  const { type, payload } = action;
+  switch (type) {
+    case ADD_BOOK:
+      return {
+        ...state,
+        isLoading: false,
+        payload,
+      };
+    case REMOVE_BOOK:
+      return {
+        ...state,
+        isLoading: false,
+        getId: state.books.filter((book) => book.id !== payload),
+      };
+    default:
+      return state;
+  }
+};
+
+// Actions
+export const addBook = (values) => (dispatch) => {
+  dispatch({
+    type: ADD_BOOK,
+    payload: values,
+  });
+};
+
+export const removeBook = (id) => (dispatch) => {
+  dispatch({
+    type: REMOVE_BOOK,
+    payload: id,
+  });
+};
+
+export default bookReducer;
```

Comment:
![image](https://user-images.githubusercontent.com/118063058/229620472-32619094-8c5b-4a3c-8327-5e802f6995b7.png)

![image](https://user-images.githubusercontent.com/118063058/229620506-7da27be5-c064-4437-b6cc-f7e6cd48f44f.png)

I tried everything that you have suggested, and I can't implement functionality that you have suggested. I saw the requirments for tomorrows project and it requires functionality that you have requested changes on, since I can not modify components in this projects.

Reply:
@enis-memic I am really sorry for all the inconveniences I have caused and also I tried reaching out again that afternoon to be on call with you but my network failed me. I have made the necessary research and I found out that the functionality is not required yet that only the redux file implementation is what you need to do
Please kindly resubmit your project and be sure the next reviewer will approve it
Am deeply sorry for the inconveniences
Thanks